exercises/zipper/zipper.js

```diff
@@ -0,0 +1,50 @@
+//
```

Comment:
Looking at https://github.com/TomPradat/javascript/blob/feature/stubs/exercises/zipper/zipper.spec.js, I don't think the methods in this file are correct. I don't see `left` and `right` for example, and things like `insertBefore` are not methods _under test.

Reply:
I get what you mean, but then there is probably a problem with the README.md for this exercise => https://github.com/exercism/javascript/blob/master/exercises/zipper/README.md

Nowhere is it mentioned that there should have `left` or `right` methods.

So the question is should we update the README.md or the example and test files ?

Reply:
Ahhh. Good question.

The README.md states

> For example given a rose tree (where each node contains a value and a list of child nodes) a zipper might support these operations:

I think we need to bolden the *For example* and we should add a JavaScript Track Hint which methods _are_ added/necessary, OR change the tests to update. I'll look into it <3

Reply:
Agreed, thanks 👍